dime-research-standards/pillar-1-research-ethics/research-ethics-resources/informed-consent-templates/informed-consent-template-verbal-covid.md

```diff
@@ -50,6 +50,11 @@ and the purpose of the disclosure.]**
 
 **[If activities are to be audio- or videotaped, specific consent for this must be obtained.]** We will record audio in some parts of the survey for ensuring data quality. Do you consent to having audio recorded during some sections of the survey ?
 
+When the research is completed, we may save the information for use in future research done by this research team.
+We will retain these records for up to **[NUMBER OF YEARS]** years after the study is over.
```

Comment:
For indefinite retention, this is the language agreed upon with the DPO: "Please note that any identifying personal data in this questionnaire will be kept until the research is completed, but safely stored and encrypted by DIME, part of the World Bank research group, so that it cannot be accessed by any third parties."

Reply:
They also asked us in the case of indefinite retention to emphasize that consent would be collected for any follow-ups, that i think is good to emphasize in all (we may already have this elsewhere in the template. Here's the approved wording from Patrick's: 

"We may also come back in the future to  do another survey, but we will ask for your   consent again, and you can decide to participate or not at that time."